app/piping/plumber.py

```diff
@@ -0,0 +1,35 @@
+import re
+
+
+class Plumber(object):
+    def __init__(self, context):
```

Comment:
With the variable name change it's definitely better, but a bit of documentation wouldn't hurt